jenkins-scripts/docker/lib/generic-abi-base.bash

```diff
@@ -46,7 +46,7 @@ cat > build.sh << DELIM
 set -ex
 
 # Bug in gcc5 with eigen see: https://github.com/ignition-tooling/release-tools/issues/147
-if [[ "${USE_GCC6}" -gt 0 || -z "${USE_GCC6}" && "${DISTRO}" == xenial ]]; then
+if [[ "${USE_GCC6}" -gt 0 || -z "${USE_GCC6}" ]]; then
```

Comment:
I think everything after the `||` should be removed:

`if [[ "${USE_GCC6}" -gt 0 ]]; then`

Reply:
The GCC6 hack was useful until Xenial. I just remove it completely in 2767f20